pkg/detectors/fastlypersonaltoken/fastlypersonaltoken.go

```diff
@@ -31,12 +31,10 @@ func (s Scanner) Keywords() []string {
 	return []string{"fastly"}
 }
 
-type fastlyUserRes struct {
-	Login                string `json:"login"`
-	Name                 string `json:"name"`
-	Role                 string `json:"role"`
-	TwoFactorAuthEnabled bool   `json:"two_factor_auth_enabled"`
-	Locked               bool   `json:"locked"`
+type token struct {
+	TokenID   string `json:"id"`
+	UserID    string `json:"user_id"`
+	ExpiresAt string `json:"expires_at"`
```

Comment:
@kashifkhan0771 Response also contains scope of that token. My suggestion would be to include it in the extraData.